recipes/bel-alex73/docker-prepare-start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
What's the goal of docker in the recipe?

Reply:
To create fully working environment to prepare and learn model.
I see that CoquiTTS has docker image for synthesis voice, but, IMHO, it would be great to have the same for learning also. Local computer has other OS, packages hell, etc. Probably, it would be good to create such docker not for our recipe only, but for full CoquiTTS.